### PR TITLE
Add fastparquet recipe

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: recipes/fastparquet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: fastparquet
+  version: "0.0.1"
+
+source:
+  fn: fastparquet-0.0.1.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.tar.gz
+  sha256: "c17b46a6f83314434143ac583968b7b0d99f331c50fae13f82e711b64a926e2a"
+
+build:
+  skip: true  # [win]
```
Comment (synapticarbors) [inline recipes/fastparquet/meta.yaml]: I don't see any indication from the `setup.py` that this should be non-Windows only. I see that `python-snappy` is non-Windows, but from the `README` this seems like an optional requirement.

File: recipes/fastparquet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas
```
Comment (synapticarbors) [inline recipes/fastparquet/meta.yaml]: Are pandas, numpy, numba and python-snappy build time requirements? It doesn't look like it, but please double check and remove if they aren't.

File: recipes/fastparquet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - numpy
+    - numba
+    - python-snappy
+  run:
+    - python
+    - pandas
+    - numpy
+    - numba
+    - python-snappy
```
Comment (synapticarbors) [inline recipes/fastparquet/meta.yaml]: `python-snappy` appears to be an optional requirement. I didn't see in the code where it is actually used so I couldn't say if it made sense to leave it to the user to install if they wanted it.

File: recipes/fastparquet/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - fastparquet
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (synapticarbors) [inline recipes/fastparquet/meta.yaml]: Incorrect `home`.

**Comment by synapticarbors** [comment]
It also looks like `requirements.txt` didn't make it into the `MANIFEST.in`. Additionally the requirements include `thriftpy`, but I didn't see it listed in the recipe.

**Comment by martindurant** [comment]
Thanks for you help, @synapticarbors and sorry for the hassle - clearly my first time doing this!

File: recipes/fastparquet/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  name: fastparquet
+  version: "0.0.1post2"
+
+source:
+  fn: fastparquet-0.0.1.post2.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.post2.tar.gz
+  sha256: "872f9849a60bbb4f10b29f76262af867ed6959209d0e5dff986bcf89bde5b57d"
+
+build:
+  number: 2
```
Comment (jakirkham) [inline recipes/fastparquet/meta.yaml]: Please set the build number to `0`.

File: recipes/fastparquet/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: fastparquet
+  version: "0.0.1post2"
+
+source:
+  fn: fastparquet-0.0.1.post2.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.post2.tar.gz
```
Comment (jakirkham) [inline recipes/fastparquet/meta.yaml]: Would recommend using jinja templates for versions to making updating more smooth. Please see the example recipe for details.

**Pull request merged**